### PR TITLE
Add news with dates & cfp announcement

### PR DESCRIPTION
**Pull request by theacodes**

### CODE REVIEW

File: content/news/2-pycascades-2020-dates-location-cfp/contents.lr [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+meta_description: PyCascades 2020 will take place on February 8th & 9th, 2020
+---
+body:
+
+We have lots of announcements to share about PyCascades 2020. 🎉🎉🎉
+
+Dates & Location
+-----------------
+
+PyCascades 2020 will take place on **Saturday, February 8th & Sunday, February 9th** at `Revolution Hall <https://www.revolutionhall.com>`__ in Portland, Oregon.
```
Comment (ericholscher) [inline content/news/2-pycascades-2020-dates-location-cfp/contents.lr]: I think this gives a bit more context for the space. 

```suggestion
PyCascades 2020 will take place on **Saturday, February 8th & Sunday, February 9th** at `Revolution Hall <https://www.revolutionhall.com/gallery/facilities/>`__ in Portland, Oregon.
```

File: content/news/2-pycascades-2020-dates-location-cfp/contents.lr [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+We'll also have our community sprints at Revolution Hall on **Monday, February 9th** as well as a pre-conference social on **Friday, February 7th**.
```
Comment (ericholscher) [inline content/news/2-pycascades-2020-dates-location-cfp/contents.lr]:
```suggestion
We'll also have our community sprints at Revolution Hall on **Monday, February 10th** as well as a pre-conference social on **Friday, February 7th**.
```

File: content/news/2-pycascades-2020-dates-location-cfp/contents.lr [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+We're not quite ready to start selling tickets yet. We'll have another announcement for that coming soon.
+Subscribe to our `newsletter </#newsletter>`__ to be the first to know (it's only important stuff, we promise)!
+
+Call for Proposals
+-------------------
+
+PyCascades has been lucky to give our stage to incredible speakers with wonderful talks over the last two years.
+We are really looking forward to showcasing our community again next year.
+Our `Call for Proposals (CFP) <https://www.papercall.io/pycascades-2020>`__ opens today and closes on **Tuesday, October 1st, 2019**.
```
Comment (ericholscher) [inline content/news/2-pycascades-2020-dates-location-cfp/contents.lr]: Timezone? AOE?

**Review by ericholscher — COMMENTED** [review]
Lots of good things! :tada:

File: content/news/2-pycascades-2020-dates-location-cfp/contents.lr (hunk 3 above)
Comment (theacodes) [reply content/news/2-pycascades-2020-dates-location-cfp/contents.lr]: Done.

**Pull request merged**